chapters/publication.tex

```diff
@@ -1,96 +1,178 @@
+and are programmed to insert code outputs in pre-specified locations.
+Documents called ``notebooks'' (such as Jupyter\sidenote{\url{https://jupyter.org/}}) work similarly,
+as they also use the underlying analytical software to create the document.
+These types of dynamic documents are usually appropriate for short or informal materials
+because they tend to offer limited editability outside the base software
+and often have limited abilities to incorporate precision formatting.
+
+The second group of dynamic document tools do not require
+direct operation of underlying code or software, but simply require
+that the writer have access to the updated outputs.
+One very simple one is Dropbox Paper, a free online writing tool
+that allows linkages to files in Dropbox,
+which are then automatically updated anytime the file is replaced.
+Dropbox Paper has very limited formatting options,
+but it is appropriate for working with collaborators who are not using statistical software.
+
+However, the most widely utilized software
+for dynamically managing both text and results is \LaTeX\ (pronounced ``lah-tek'').\sidenote{
+  \url{https://www.maths.tcd.ie/~dwilkins/LaTeXPrimer/GSWLaTeX.pdf}}
+  \index{\LaTeX}
+Rather than using a coding language that is built for another purpose
+or trying to hide the code entirely,
+\LaTeX\ is a special code language designed for document preparation and typesetting.
+While this tool has a significant learning curve,
+its enormous flexibility in terms of operation, collaboration,
+and output formatting and styling
+makes it the primary choice for most large technical outputs today,
+and it has proven to have enduring popularity.
+In fact, \LaTeX\ operates behind-the-scenes in many of the tools listed before.
+Therefore, we recommend that you learn to use \LaTeX\ directly
+as soon as you are able to and provide several resources for doing so.
+
+\subsection{Technical writing with \LaTeX}
+
+\LaTeX\ is billed as a ``document preparation system''.
+What this means is worth unpacking.
+In {\LaTeX}, instead of writing in a ``what-you-see-is-what-you-get'' mode
+as you do in Word or the equivalent,
+you write plain text interlaced with coded instructions for formatting
+(similar in concept to HTML).
+The \LaTeX\ system includes commands for simple markup
+like font styles, paragraph formatting, section headers and the like.
+But it also includes special controls for including tables and figures,
+footnotes and endnotes, complex mathematical notation, and automated bibliography preparation.
+It also allows publishers to apply global styles and templates
+to already-written material, allowing them to reformat entire documents in house styles
+with only a few keystrokes.
+In sum, \LaTeX\ enables automatically-organized documents,
+manages tables and figures dynamically,
+and because it is written in a plain text file format,
+\texttt{.tex} can be version-controlled using Git.
```

Comment:
this point should be made earlier (not just in conclusion)

Reply:
408f6c0fe400129ac1140abf725bd61afbee7adb